index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <title>Portfolio - Bright Kweku Ahiadeke</title>
+      <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header>
+      <nav class="header-nav wrap">
+        <h1 class="logo-text">Bright</h1>
+        <span>
+          <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z" fill="#6070FF"/>
+          </svg>
+        </span>
+      </nav>
+    </header>
+
+    <section class="headline">
+      <div class="headline-wrap">
+        <h2 class="welcome">I’m Bright Glad to see you!</h2>
+        <p class="headline-profile margin-top-12">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="headline-cta margin-top-12">Let's connect</a>
+        <ul class="social-icons margin-top-12">
+          <li>
+            <a>
+              <img class="social twitter" src="images/icons/twitter-icon.png" alt="Twitter icon">
```

Comment:
- [ ] [REQUIRED] I think it would be great if you add a dead link `href='#'` attribute to all the `<a>` tags for the different social icons to make them clickable 🔗